pandas/core/groupby/ops.py

```diff
@@ -790,7 +790,7 @@ def _get_axes(group):
         return group.axes
 
 
-def _is_indexed_like(obj, axes):
+def _is_indexed_like(obj, axes) -> bool:
```

Comment:
I think reasonable to annotate `obj` as FrameOrSeries here, unless the TypeVar doesn't work in which case a Union